io.github.mak448a.QTCord.yml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.mak448a.QTCord
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+command: io.github.mak448a.QTCord
+finish-args:
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+modules:
+  - python3-modules.json
+  - name: BuildQTCord
+    buildsystem: simple
+    build-commands:
+      - install -D src/flatpak_builder/io.github.mak448a.QTCord /app/bin/io.github.mak448a.QTCord
+      - mv licenses/ /app/bin/
+      - install -Dm 644 io.github.mak448a.QTCord.metainfo.xml /app/share/metainfo/io.github.mak448a.QTCord.metainfo.xml
```

Comment:
Swicth it to upstream https://github.com/mak448a/QTCord/blob/main/src/flatpak_builder/io.github.mak448a.QTCord.metainfo.xml

Reply:
How do I include the file?
Like this?

```
sources:
  - type: file
        url: https://github.com/mak448a/QTCord/blob/main/src/flatpak_builder/io.github.mak448a.QTCord.metainfo.xml
```

Reply:
`install -Dm 644 src/flatpak_builder/io.github.mak448a.QTCord.metainfo.xml /app/share/metainfo/io.github.mak448a.QTCord.metainfo.xml`

Reply:
You have to update that file first here https://github.com/mak448a/QTCord/blob/main/src/flatpak_builder/io.github.mak448a.QTCord.metainfo.xml and make a new tag.

Reply:
Ah ok